bumble/core.py

```diff
@@ -144,9 +144,12 @@ def __init__(
 class UUID:
     '''
     See Bluetooth spec Vol 3, Part B - 2.5.1 UUID
+
+    Note that this class expects and works in little-endian byte-order throughout.
+    The exception is when interacting with strings, which are in big-endian byte-order.
     '''
 
-    BASE_UUID = bytes.fromhex('00001000800000805F9B34FB')
+    BASE_UUID = bytes.fromhex('00001000800000805F9B34FB')[::-1] # little-endian
```

Comment:
I suggest putting it directly in little endian rather than converting: "BF43B9F50800000800010000"

Reply:
I would prefer the current solution, since it's the same has in the spec it's easier to understand

Reply:
Makes sense. Let's keep it like this then.